deel-2-flexbox-layout/opdracht-4/styles/styles.css

```diff
@@ -1 +1,51 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body {
+    display: flex;
+    height: 100vh;
+}
+
+.outer-col-1 {
+    display: flex;
+    flex-basis: 400px;
+}
+
+.outer-col-2 {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 2000px;
+}
+
+.inner-row {
+    display: flex;
+    flex-basis: 1500px;
+}
```

Comment:
toevoegen: flex-direction: row (omdat ie nu in een column van outer-col-2 staat moet je dat opnieuw defineren als dat anders moet zijn)